static/src/javascripts/bootstraps/enhanced/notifications.js

```diff
@@ -148,6 +108,10 @@ define([
                     } else {
                         return reg.pushManager.subscribe({userVisibleOnly: true});
                     }
+                }).catch(function(e) {
+                    var $denied = $('.live-notifications__label--denied');
+                    $denied.show('block');
```

Comment:
Would use an  `is-visible` modifier class for styling here too and `js-` class for js